src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -3491,6 +3515,21 @@ operand no_rax_rdx_RegI()
   interface(REG_INTER);
 %}
 
+operand no_rbp_r13_RegI()
+%{
+  constraint(ALLOC_IN_RC(int_no_rbp_r13_reg));
+  match(RegI);
+  match(rRegI);
+  match(rax_RegI);
```

Comment:
Why explicitly match rax, rbx, ..., rdi here?

Reply:
Tbh I'm not sure but the other operands seem to do the same thing